graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -167,13 +234,16 @@ def rename_schema(
         type_renamings: maps original type name to renamed name or None (for type suppression). A
                         type named "Foo" will be unchanged iff type_renamings does not map "Foo" to
                         anything, i.e. type_renamings.get("Foo", "Foo") returns "Foo"
+        field_renamings: maps type names to the renamings for its fields. The renamings map field
+                         names belonging to the type to a set of field names for the renamed
+                         schema
```

Comment:
Hmm, I can see why it'd be a bit tricky. The original comment avoids using the word "dictionary" because the renamings could also just be an object implementing the `get` method. I also tried separating the type from its name `type_name`, so I'm not quite sure how to approach changing this one on either front without losing accuracy.